common/config/rush/command-line.json

```diff
@@ -33,6 +33,14 @@
       "shellCommand": "rush install && cpy --flat common/config/env/beta.env common/config/env --rename .env && node common/scripts/invalidate-build.js",
       "safeForSimultaneousRushProcesses": true
     },
+    {
+      "commandKind": "global",
```

Comment:
Happy to include these if multiple folks will find this useful - but we need to add documentation along side if we do.
The alternative is to simply have a local bash/powershell profile with aliases specified

Reply:
> I thought about have a local bash profile but it would be nice to have automatically in the repo. Then it would work in codespaces etc with no additional changes.

Are you thinking documentation here?

https://github.com/Azure/communication-ui-library/tree/main/docs/contributing-guide

Reply:
Added